CMakeLists.txt

```diff
@@ -377,6 +377,20 @@ if(FLB_HAVE_CLOCK_GET_TIME)
   FLB_DEFINITION(FLB_HAVE_CLOCK_GET_TIME)
 endif()
 
+# unix socket support
+check_c_source_compiles("
+  #include <sys/types.h>
+  #include <sys/socket.h>
+  int main() {
+      int sock;
+      sock = socket(AF_INET, SOCK_STREAM, 0);
```

Comment:
it should be AF_UNIX, otherwise its creating a normal TCP socket.

Reply:
Oops. I've fixed it.